motion/thrust_merger/src/thrust_merger.cpp

```diff
@@ -58,6 +59,7 @@ void ThrustMerger::spin()
     geometry_msgs::Wrench wrench_msg;
     tf::wrenchEigenToMsg(combined_wrench, wrench_msg);
     thrust_pub.publish(wrench_msg);
+    thrust_pub_sim.publish(wrench_msg);
 
```

Comment:
I don't think this is the best approach to getting thrust to the simulator, since we end up publishing wrench twice, even when not running the simulator. A better way to handle this will be to add a [remap tag](http://wiki.ros.org/roslaunch/XML/remap) only when launching the simulator, as this can be done from the top-level launch, or motion.launch.